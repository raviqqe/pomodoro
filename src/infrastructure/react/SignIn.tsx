import React from "react";
import { TextButton } from "./TextButton";

interface IProps {
  signIn: () => void;
}

export const SignIn = ({ signIn }: IProps) => (
  <TextButton onClick={signIn}>Sign in</TextButton>
);
