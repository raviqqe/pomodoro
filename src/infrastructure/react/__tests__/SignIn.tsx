import { create } from "react-test-renderer";
import React from "react";
import { SignIn } from "../SignIn";

it("renders", () => {
  expect(
    create(<SignIn signIn={() => undefined} />).toJSON()
  ).toMatchSnapshot();
});
