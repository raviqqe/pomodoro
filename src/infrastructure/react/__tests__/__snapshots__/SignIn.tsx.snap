// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<button
  className="sc-bdVaJa sc-bwzfXH cqtuLn"
  onClick={[Function]}
>
  Sign in
</button>
`;
